CMakeLists.txt

```diff
@@ -38,17 +40,50 @@ if(ARGH_MASTER_PROJECT)
 	install(TARGETS argh EXPORT arghTargets)
 
 	include(GNUInstallDirs)
+	string(REPLACE "/${CMAKE_LIBRARY_ARCHITECTURE}" "" CMAKE_INSTALL_LIBDIR_ARCHIND "${CMAKE_INSTALL_LIBDIR}")
 	install(FILES "${CMAKE_CURRENT_LIST_DIR}/argh.h" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 	install(FILES "${CMAKE_CURRENT_LIST_DIR}/LICENSE" DESTINATION ${CMAKE_INSTALL_DOCDIR})
 	install(FILES "${CMAKE_CURRENT_LIST_DIR}/README.md" DESTINATION ${CMAKE_INSTALL_DOCDIR})
+	set(PackagingTemplatesDir "${CMAKE_CURRENT_LIST_DIR}/packaging")
 
 	if(CMAKE_SYSTEM_NAME STREQUAL Linux)
 	# this might be a bit too restrictive, since for other (BSD, ...) this might apply also
 	# but this can be fixed later in extra pull requests from people on the platform
-		install(FILES argh-config.cmake DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/argh)
-		install(EXPORT arghTargets DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/argh)
+		install(FILES "${PackagingTemplatesDir}/argh-config.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR_ARCHIND}/cmake/argh)
```

Comment:
This change might require changes in other package managers, that will need to adapt to the new install layout. As long as changes come in a new version, it is doable, but extra effort to implement by other package managers, which logic might become dirtier:

```
if argh-version < x.y.z:
     files installed in cmake/argh
else
     files installed in arch/cmake/argh
```

Reply:
>This change might require changes in other package managers, that will need to adapt to the new install layout.

IDK if really. Debian/Ubuntu, Fedora and Arch already support this layout, so it is not "new". IDK about other distros.